tests/integration/docarray_v2/sagemaker/test_reranking.py

```diff
@@ -0,0 +1,112 @@
+import csv
+import io
+import os
+from contextlib import AbstractContextManager
+
+import requests
+
+from jina.orchestrate.pods import Pod
+from jina.parsers import set_pod_parser
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+sagemaker_port = 8080
+
+
+class chdir(AbstractContextManager):
+    def __init__(self, path):
+        self.path = path
+        self._old_cwd = []
+
+    def __enter__(self):
+        self._old_cwd.append(os.getcwd())
+        os.chdir(self.path)
+
+    def __exit__(self, *excinfo):
+        os.chdir(self._old_cwd.pop())
+
+
+def test_provider_sagemaker_pod_inference():
+    with chdir(os.path.join(os.path.dirname(__file__), "SampleRerankerExecutor")):
```

Comment:
u do not need this, u can pass the path to the `config.yml` without this

Reply:
fixed